packages/mux-player-react/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "outDir": "dist",
     "incremental": true,
     "target": "ES2019",
-    "module": "es6",
+    "module": "ES2020",
```

Comment:
This seems like a big change that shouldn't be missed.
ES2020 added dynamic import support, necessary for React.lazy

Reply:
question(blocking): Have you confirmed this is doing what you expect? FYI we use `esbuild` (and not directly using `tsc`) for our build targets, so I'm honestly not sure how that relationship shakes out.

Reply:
I can at least confirm the build fails without it! During the  `build:types` phase:
```
@mux/mux-player-react:build: src/lazy.tsx(4,41): error TS1323: Dynamic imports are only supported when the '--module' flag is set to 'es2020', 'es2022', 'esnext', 'commonjs', 'amd', 'system', 'umd', 'node12', or 'nodenext'.
```

Reply:
ah that tracks since we use `tsc` specifically for type def generation.

Reply:
might want to confirm that the dist looks like you'd expect as well. You can muck around with our esbuild script usage locally to e.g. not minify if that makes things easier.